assets/stylesheets/dashboard.scss

```
@@ -96,6 +96,14 @@ h2 .fa {
     font-size: 90%;
 }
 
+.build-row .fas {
+    visibility:hidden;
```

please add spaces here for better readability.
```suggestion
    visibility: hidden;
```